shell/platform/windows/win32_flutter_window.cc

```diff
@@ -1,6 +1,10 @@
 #include "flutter/shell/platform/windows/win32_flutter_window.h"
-
 #include <chrono>
+#include "flutter/shell/platform/common/cpp/client_wrapper/include/flutter/json_message_codec.h"
+
+static constexpr char kSystemChannelName[] = "flutter/system";
```

Comment:
win32_flutter_window doesn't seem like the right place for the flutter/system channel to live. Please set up a new handler for now, following the model of platform_handler.{h,cc}. Later we may decide to fold it into another class, but that will be easier if it's not embedded into this one to start with.

Reply:
This is all cruft now.